docs/engine.md

```diff
@@ -5,3 +5,8 @@
         show_root_heading: true
         docstring_section_style: table
         show_signature_annotations: false
+        members:
```

Comment:
you could try: `members_order: source`

Reply:
Current downside is that you have the list written here instead of relying on the code